indigo_api/views/attachments.py

```diff
@@ -51,6 +68,13 @@ def view(self, request, *args, **kwargs):
         attachment = self.get_object()
         return view_attachment(attachment)
```

Comment:
What about moving the PDF logic here? The idea being, if the document isn't already a PDF, convert it to a pdf. The UI only uses `../view` to show the file, so converting to a PDF is safe. The `../download` is used to actually download it.

It would mean the UI can always ask for `.../view` and know that it'll get back something the browser can show (ie. a pdf)

Reply:
It should mean that the bulk of the JS changes below fall away.